modules/geopackage/src/lib/parse-geopackage.ts

```diff
@@ -261,7 +261,7 @@ function constructGeoJsonFeature(
  * @param db database
  * @returns version string. One of '1.0', '1.1', '1.2'
  */
-// eslint-disable-next-line no-unused-vars
+/* eslint-disable @typescript-eslint/no-unused-vars */
```

Comment:
Agree that eslint-disable-line and eslint-disable-next-line are preferable for occasional use, rather than disabling for full file. One exception is `camelcase` as some files deal with data formats that use camelcase and it is better to just disable the entire file.

For the unused parameters, let's try to add this subrule: https://eslint.org/docs/rules/no-unused-vars#args-after-used

The problem is that the parser function are expected to be standardized across loaders so I prefer not to drop arguments that are expected by the loaders.gl architecture.

Reply:
> For the unused parameters, let's try to add this subrule: https://eslint.org/docs/rules/no-unused-vars#args-after-used
> 
> The problem is that the parser function are expected to be standardized across loaders so I prefer not to drop arguments that are expected by the loaders.gl architecture.

Yes, I agree, this rule is more useful for those kind of situations